packages/suite/src/actions/suite/analyticsActions.ts

```diff
@@ -33,14 +33,8 @@ export type AnalyticsAction =
           };
       };
 
-/**
-simple semver for data-analytics part.
-<breaking-change>.<analytics-extended>
-
-Don't forget to update docs with changelog!
-*/
-
-export const version = '1.8';
+//  Don't forget to update docs with changelog!
+export const version = '1.9'; // <breaking-change>.<analytics-extended>
```

Comment:
I tend to always write comments on a separate line rather than to append them to the code. It imo reads nicer and doesn't risk being out of viewport in GUI.

```suggestion
 // <breaking-change>.<analytics-extended>
 export const version = '1.9';
```